### ISSUE TITLE
Altstore and installed apps are invalid after refresh or installation

### ISSUE THREAD

**Issue by Hycade** [rileytestut/AltStore#1066]
Issue - After refreshing apps altstore and all installed apps will become unavailable, this is also possible through installing the same app from different libraries (Ex: 2 different YouTube IPA’s). Reinstalling altstore without removing it and all apps from the device will cause altserver to completely freeze and requires task manager to close it. 

Info - Phone: 
iPhone 11 
IOS 16.1 
Free developer account
Alt-store/Server version - 1.5.1

**Comment by Thunderz2016** [comment]
Can confirm this issue on iOS 16.1 RC (20B79) with a free developer account. However, simply refreshing does not make the sideloaded apps unavailable for me, it is installing a new app that does. Once my device is restored to iOS 16.0.3 the issue is resolved.

**Comment by jun6lee** [comment]
Seeing the same behaviour as above described for iPadOS 16.1 after upgrade. Tried removing AltServer (+sideloaded apps) and re-installing it.

**Comment by shanegillio** [comment]
are you still seeing this issue as of 16.2? wondering if this is limited to 16.1

**Comment by Thunderz2016** [comment]
From my testing this issue is still present on iOS 16.3 Public Beta 2 (20D5035i) with both AltStore 1.5.1 and 1.6rc.

**Comment by velaar** [comment]
That is correct. Installing any app breaks all including Alt Store. Reinstalling AltStore makes the apps available until the next install or refresh.

**Comment by shanegillio** [comment]
Question - is this actually ANY app? From lots of testing, I have found that downloading apps from a source or store page in AltStore does not cause issues. However, is this maybe related to apps sideloaded with the '+' icon?
